package.json

```diff
@@ -1,29 +1,53 @@
 {
   "name": "@loke/http-rpc-client",
-  "version": "3.0.1",
+  "version": "3.1.0-beta.0",
```

Comment:
Any reason why beta?

Just want to check it all works as a package first?

Reply:
Yeah was going to bump it once reviewed